tests/testthat/examples/interpolate/orderly.R

```diff
@@ -0,0 +1,7 @@
+orderly3::orderly_parameters(mpg_min = NULL)
+orderly3.db::orderly_db_query(
+  query = "SELECT * FROM mtcars WHERE mpg > ?mpg_min",
```

Comment:
Nice! Worth documenting this `?name` syntax somewhere? Maybe in `orderly_db_query` and `orderly_db_view`?

Reply:
yeah - this is from the original orderly, I'll get some docs transferred over